lib/api/websiteGet.js

```diff
@@ -12,13 +12,14 @@ import { isObjAuthorized } from './apiUtils/authorization/aclChecks';
 import collectResponseHeaders from '../utilities/collectResponseHeaders';
 import validateHeaders from '../utilities/validateHeaders';
 import { pushMetric } from '../utapi/utilities';
+import { isBucketAuthorized } from './apiUtils/authorization/aclChecks';
 
 /**
  * _errorActions - take a number of actions once have error getting obj
  * @param {object} err - arsenal errors object
  * @param {string} errorDocument - key to get error document
  * @param {object []} routingRules - array of routingRule objects
- * @param {string} bucketName - bucket name from request
+ * @param {string} bucket - bucket from request
```

Comment:
this is an object, not a string, correct?
I would describe it as the bucket metadata, not bucket from request.